examples/ios-beta/readme.md

```diff
@@ -7,7 +7,7 @@ Since this needs to run in an iOS simulator or on a device, you can't run it lik
 - Start a simulator (Simulator.app).
 - `cargo install cargo-bundle`
 - `cargo bundle --example ios-beta --no-default-features --features uikit,autolayout --target x86_64-apple-ios`
```

Comment:
Shall we add this to the github runner?

Reply:
Probably makes sense to do so, yeah - that way we'll know if I break it again ;)